incubator-tools/importing_processor_and_evaluating_with_alternate_test_sets/importing_processor_and_evaluating_with_alternate_test_sets.ipynb

```diff
@@ -0,0 +1,766 @@
+{
```

Comment:
<p>A lot of the code in this notebook can be simplified/cleaned up. For example, it could make sense to put all of the wrapper/utility functions at the top of the notebook or in the utilities.py file</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
The functions which ever are in this notebook are available in document AI API documentation , we have used the same  and few which are not available are already added in utilities.py file. This document is to guide how to use the existing functions for getting the evaluation of processor using files in GCS folder .